src/lib/containers/entity/annotations/AdditionalPropertiesSchemaField.tsx

```diff
@@ -135,8 +135,9 @@ export function AdditionalPropertiesSchemaField<T>(
   } = useListState(
     Array.isArray(formData)
       ? formData
-      : // Coerce individual value to an array
-        [formData],
+      : typeof formData === 'string'
+      ? formData.split(',').map(s => s.trim()) // split a string of comma-separated values, then trim whitespace
+      : [formData], // Coerce individual value to an array
```

Comment:
Add another branch for converting a schema-defined field to an additional property: if it's a string, split at commas.

The inverse happens in CustomDefaultTemplate.tsx, so converting between schema-defined string and custom annotation array is lossless

Reply:
Got it.  I've struggled with stringing these ternary operators together.  I think once is fine, twice is not ideal, and greater than or equal to three times makes it too difficult to read.